packages/data-mate/src/function-configs/geo/geoIntersects.ts

```diff
@@ -0,0 +1,106 @@
+import { geoIntersectsFP, toGeoJSON } from '@terascope/utils';
+import { FieldType, GeoInput, GeoShapeType } from '@terascope/types';
+import {
+    FieldValidateConfig,
+    ProcessMode,
+    FunctionDefinitionType,
+    FunctionDefinitionCategory,
+    FunctionDefinitionExample
+} from '../interfaces';
+
+export interface GeoContainsArgs {
+    geoInput: GeoInput;
```

Comment:
Same as `geoContains`, this should probably be `value`

```suggestion
    value: GeoInput;
```